### PR TITLE
fixed module multiplestringliterals

### PR DESCRIPTION
**Pull request by 555vedant**
# PLEASE READ before removing

Rules:
1) The issue you are trying to fix/resolve has to have the "approved" label.
2) Put in the description of Pull Request the reference to an issue if it exists.
   Example: "Issue: #XXXXXX"
3) Commit message should adhere to the following rules:
   a) MUST match one of the following patterns:
      ^Issue #\\d+: .*$
      ^Pull #\\d+: .*$
      ^(minor|config|infra|doc|spelling|dependency): .*$
   b) MUST contain only one line of text
   c) MUST NOT end with a period, space, or tab
   d) MUST be less than or equal to 200 characters

To avoid multiple iterations of fixes and CIs failures, please read
https://checkstyle.org/contributing.html

ATTENTION: We are not merging Pull Requests that are not passing our CIs,
but we will help to resolve issues.

Thanks for reading, remove whole this message and type what you need.

### CODE REVIEW

**Comment by 555vedant** [comment]
sir why we compulsory need of CIRRUS  ?

File: src/xdocs/checks/coding/multiplestringliterals.xml [hunk 1]
```diff
@@ -74,11 +74,11 @@
 public class MyClass {
   String a = &quot;StringContents&quot;;
   String a1 = &quot;unchecked&quot;;
-  @SuppressWarnings(&quot;unchecked&quot;) // OK, duplicate strings are ignored in annotations
```
Comment (MANISH-K-07) [inline src/xdocs/checks/coding/multiplestringliterals.xml:77]: Please go through the issue #13213 description again.

**Review by MANISH-K-07 — COMMENTED** [review]
Please try to refer previously merged PRs and understand the requirement of the issue.

File: src/xdocs/checks/coding/multiplestringliterals.xml (hunk 1 above)
Comment (555vedant) [reply src/xdocs/checks/coding/multiplestringliterals.xml:77]: @MANISH-K-07  in line 77 comment making sense regarding duplicate string  why we need to remove that commnet?

Comment (MANISH-K-07) [reply src/xdocs/checks/coding/multiplestringliterals.xml:77]: Exactly my question !!
Why did you remove such sensible coments ??

The issue only asks you to remove unnecessary '//ok' comments which don't have any explanation followed,  from `Input files`. Please find the right path to the suppressed file.

**Review by MANISH-K-07 — COMMENTED** [review]
Again, please look at previously merged PRs and issue description to understand the requirement.

I have attached a few examples of updates : 

- https://github.com/checkstyle/checkstyle/pull/14258
- https://github.com/checkstyle/checkstyle/pull/14259
- https://github.com/checkstyle/checkstyle/pull/14260

**Pull request closed without merging**